ci-operator/step-registry/upi/install/vsphere/upi-install-vsphere-ref.yaml

```diff
@@ -18,6 +18,10 @@ ref:
     name: ci-route-53
     mount_path: /var/run/vault/vsphere-aws
   env:
+  - name: FIRMWARE_SET
+    default: "efi"
```

Comment:
suggest to set default value as "", so that we can always use fireware from rhcos base image if env FIRMWARE_SET is not set.

Reply:
ok.got it.